Can't upgrade Downshift

Can anyone update `downshift` in `@wordpress/components`? I've tried through all possible methods, but using `4.0.5` always leads to the build failing with:

```sh
Module not found: Error: Can't resolve 'downshift' in '/Users/epiqueras/Desktop/Work/gutenberg/packages/components/src/custom-select-control'
```

--- a/packages/components/package.json
+++ b/packages/components/package.json
@@ -42,7 +42,7 @@
 		"classnames": "^2.2.5",
 		"clipboard": "^2.0.1",
 		"dom-scroll-into-view": "^1.2.1",
-		"downshift": "^3.3.4",
+		"downshift": "^4.0.5",
 		"gradient-parser": "^0.1.5",
 		"lodash": "^4.17.15",
 		"memize": "^1.0.5",

